### PR TITLE
Add NelmioApiDocBundle recipe

### PR DESCRIPTION
**Pull request by 20uf**
| Q             | A
| ------------- | ---
| License       | MIT

About this bundle:
The most popular one for generate a decent documentation for your APIs.
6 million downloads
https://github.com/nelmio/NelmioApiDocBundle
@GuilhemN as the author of the bundle

### CODE REVIEW

File: nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+nelmio_api_doc: ~
```
Comment (GromNaN) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml]: File name should be `nelmio_api_doc.yaml`

File: nelmio/api-doc-bundle/2.13/etc/routing/nelmio_api_doc_routing.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+NelmioApiDocBundle:
```
Comment (GromNaN) [inline nelmio/api-doc-bundle/2.13/etc/routing/nelmio_api_doc_routing.yaml]: The name should be `nelmio_api_doc`. See the web_profiler: https://github.com/symfony/recipes/blob/master/symfony/web-profiler-bundle/3.3/etc/routing/dev/web_profiler.yaml

File: nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml (hunk 1 above)
Comment (Pierstoval) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml]: And by the way, maybe add some default values or comments about values that may be change by the end-developer is a good idea. Having an almost-empty file is not really a good practice

File: nelmio/api-doc-bundle/2.13/manifest.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Nelmio\\ApiDocBundle\\NelmioApiDocBundle": ["all"]
```
Comment (Pierstoval) [inline nelmio/api-doc-bundle/2.13/manifest.json]: Is it common to have ApiDoc available on prod too?

I have never used it manually, but on projects I used to work on in the past, NelmioApiDocBundle was enabled only in dev mode 🤔

**Comment by fabpot** [comment]
@20uf Can you take into account the comments from the reviewers? Thanks.

**Comment by 20uf** [comment]
@fabpot Yes, from the return of my holidays I take care of it! Sorry for delays.

File: nelmio/api-doc-bundle/2.13/manifest.json (hunk 3 above)
Comment (20uf) [inline nelmio/api-doc-bundle/2.13/manifest.json]: I am fifty/fifty on this issue, an API can be public and documentation open.
Also in the [official documentation ](https://symfony.com/doc/current/bundles/NelmioApiDocBundle/index.html)of the bundle is declared for production.

File: nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml (hunk 1 above)
Comment (20uf) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml]: @Pierstoval I approve of enriching this point, but it requires maintenance on this repository if there are changes on the reference configurations of the project.
The question is: is it not wiser to comment only on the link to the documentation?

File: nelmio/api-doc-bundle/2.13/etc/routing/nelmio_api_doc_routing.yaml (hunk 2 above)
Comment (20uf) [inline nelmio/api-doc-bundle/2.13/etc/routing/nelmio_api_doc_routing.yaml]: Done

File: nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml (hunk 1 above)
Comment (20uf) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml]: @GromNaN Done, thanks.

**Comment by 20uf** [comment]
@Seldaek Can you do a review on this PR and give your opinion? Thank you.

**Comment by Seldaek** [comment]
I am not the main author nor have I used or maintained this bundle in years so I don't think I'm the best person..

**Comment by 20uf** [comment]
Ho ok sorry. @GuilhemN ?

**Comment by GuilhemN** [comment]
Imo we should only keep the `name` and the `sandbox` sections, the others aren't that common.

**Comment by 20uf** [comment]
Ok done @GuilhemN :+1:

File: nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+nelmio_api_doc: ~
```
Comment (fabpot) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc.yaml]: What about removing `~` here? I think it's not strictly needed and avoid errors if one only uncomment entries below.

Comment (20uf) [inline nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc.yaml]: I agree with your opinion. Done.

**Pull request merged**